cmd/ak/common/flags.go

```diff
@@ -68,3 +69,19 @@ func ToExitCodeWithSkipNotFoundFlag(cmd *cobra.Command, err error, whats ...stri
 	}
 	return exitErr
 }
+
+// maybe we could just run this check for all required flags instead of providing explicit ones?
+func UnsetEmptyFlags(cmd *cobra.Command, flags ...string) {
+	flagsMap := make(map[string]struct{}, len(flags))
+	for _, flag := range flags {
+		flagsMap[flag] = struct{}{}
+	}
+
+	cmd.Flags().Visit(func(flag *pflag.Flag) {
+		if _, ok := flagsMap[flag.Name]; ok {
+			if flag.Value.Type() == "string" && flag.Changed && flag.Value.String() == "" {
+				flag.Changed = false
```

Comment:
don't we want to reset the value as well?

Reply:
no, because the value in both cases is `""` e.g. empty string.
But in one it's a default value for argument and in the other it's the value passed by the user